tutorials/how-to-setup-applesilicon-server-with-terraform-ansible/index.mdx

```diff
@@ -0,0 +1,229 @@
+
+### Ansible
+[Ansible](https://www.ansible.com/) is an automation tool that simplifies configuration management, application deployment, and task automation. By using playbooks, Ansible helps automate and streamline processes, ensuring consistency and efficiency in managing complex infrastructure.
+
+## How to create an apple-silicon server
+
+### Using Terraform
+
+1. Download and install [Terraform](https://developer.hashicorp.com/terraform/install)
+
+2. Create a Directory: First, create a directory for your Terraform project. Open your terminal and run:
+
+```shell
+mkdir apple_silicon_server_terraform
+cd apple_silicon_server_terraform
+```
+
+3. Create a Terraform Configuration File: Inside this directory, create a file named resources.tf:
+
+```shell
+touch resources.tf
+```
+
+4. Define the Required Providers: Open the resources.tf file and add the following configuration to define the Scaleway provider and set the required Terraform version:
+
+```shell
+terraform {
+  required_providers {
+    scaleway = {
+      source = "scaleway/scaleway"
+    }
+  }
+  required_version = ">=0.13"
+}
+
+```
+
+5. Define the Apple Silicon Server: Add the following code to define your Apple Silicon server (M2-M type) in the same resources.tf file:
+
+```terraform
+#resources.tf
+resource "scaleway_apple_silicon_server" "server" {
+    name = "MyAwesomeServer"
+    type = "M2-M"
+    zone = "fr-par-1"
+}
+```
+
+6. Apply the Configuration: To apply this configuration, run the following commands in your terminal:
+
+```shell
+#Initialize Terraform
+terraform init
+#Plan the deployment
+terraform plan
+#Create the server
+terraform apply
+```
+
+When prompted, type yes to confirm the creation of the resources.
+
+7. Enable Virtual Private Cloud (VPC) and Private Network: To enhance the network setup, you can update the configuration to enable VPC option and attach a private network to your Apple Silicon server. Update your resources.tf file with the following
+
+```terraform
+#resources.tf
+resource "scaleway_vpc" "vpc01" {
+    name = "MyAwesomeVPC"
+}
+
+resource "scaleway_vpc_private_network" "pn01" {
+    name = "MyAwesomePN"
+    vpc_id = scaleway_vpc.vpc01.id
+}
+
+resource "scaleway_apple_silicon_server" "server" {
+    name = "MyAwesomeServer"
+    type = "M2-M"
+    zone = "fr-par-1"
+    enable_vpc = true
+    private_network {
+          id = scaleway_vpc_private_network.pn01.id
+    }
+}
+```
+8. Apply the Configuration Update: Finally, run the following command to apply the changes and update the server configuration
+
+```shell
+terraform apply
+```
+
+This will apply the new settings, ensuring that the server is launched within the specified VPC and connected to the private network.
+
+### Using Ansible
+
+1. Install Ansible:  If you don't have Ansible installed on your system, you can easily install it using pip, the Python package manager. Run the following command:
```

Comment:
I would simply put a link to the ansible documentation as it can cover more up to date information and many platforms